content/2022/prepoznavanje-govora.md

```diff
@@ -2,3 +2,231 @@
 title: Prepoznavanje govora
 summary: Projekat iz prepoznavanja govora rađen na letnjem kampu za stare polaznike 2022. godine od Dimitrija Pešića i Lazara Zubovića.
 ---
+### Apstrakt
+
+### Apstrakt na engleskom
+### Uvod
+
+Projekat "Prepoznavanje govora" pomaže pri rešavanju popularne dileme u AI tehnologiji, a to je kako da se glas pretvori u kucani tekst. Motivacija projekta bila je u tome da se ne samo primene mnoge metode korišćene za speech recognition, već da se i uporede njihova praktičnost i upotreba u praktičnim svrhama. 
+
+Projekat se zasniva na ideji korišćenja spektrograma kao osnovne metode prikaza zvuka u 2D formatu. Iz tog formata, drugačijim metodama bi se zvuk prepoznavao sa spektrograma što je zapravo ništa drugo no obična slika. Sa te slike mogu se pokupiti različiti podaci o zvuku zarad preciznijeg prepoznavanja istog.
+
+Prelazeći kroz literaturu i referentne radove, mnogi su više doprineli pri metodi obrade spektrograma nego pri izradi samih spektrograma. 
+
+Osvrt na rad ogleda se u metodama koje su pokrivene u referentnim radovima, poput MFCC-a (Mel-Frequency Cepstral Coefficients), logističke regresije, Random Forest-a, SVM-a, XGBoost-a, kao i konvolucionih neuronskih mreža. Do ovog projekta, ljudi su fokusirali svoje radove na obradi jednog metoda i testiranju na određenoj bazi. Nasuprot njihovim, ovaj rad ima dosta limitiranu bazu, te i sami rezultati variraju u odnosu na već dobijene.
+### Aparatura i metoda
```

Comment:
Mislim da je nepotrebno da ovaj naslov sadrži i reč "aparatura", s obzirom da je vi, u suštini, nemate (osim ako ne planirate da posvetite ovaj deo tome da navodite aparaturu korišćenu za deo projekta u kom ste vi snimali podatke za svoju bazu podataka). Takođe, bitno nam je da imamo uvid u to koji su podaci korišćeni, pa je skroz okej pre celog ovog dela dodati i segment "Podaci" ili nešto slično gde ćete govoriti o obe baze podataka koje ste koristili: nad čime ste vi zapravo radili prepoznavanje govora, koliko podataka imamo, ...